include/ImageGenerator.h

```diff
@@ -0,0 +1,31 @@
+#include <vector>
+#include <algorithm>
+#include <functional>
+#include <opencv2/core.hpp>
+#include <opencv2/imgproc.hpp>
+#include <opencv2/highgui.hpp>
+#include <opencv2/imgproc.hpp>
+#define w 400
+using std::vector;
+using namespace cv;
+
+char hull_window[] = "Drawing: Convex Hull";
```

Comment:
Esto no sirve cambielo a define
```suggestion
#define hull_window "Drawing: Convex Hull";
```